sdk/tests/deqp/functional/gles3/es3fPixelBufferObjectTest.js

```diff
@@ -287,6 +287,11 @@ var tcuImageCompare = framework.common.tcuImageCompare;
                 break;
         }
 
+        if (gl.checkFramebufferStatus(gl.FRAMEBUFFER) != gl.FRAMEBUFFER_COMPLETE) {
```

Comment:
There are definitely framebuffer attachment formats that are required to result in a complete framebuffer. It looks to me like this could allow the implementation to behave badly and report them all as incomplete. Do you agree? If so, could you modify the tests so they have a notion of whether they're optional or not?